src/Integration.Vsix/BoundSolutionAnalyzer.cs

```diff
@@ -28,8 +28,6 @@
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
-    // TODO: CM2: update to record telemetry for the new connected mode
```

Comment:
We discussed this offline with the PM and decided we didn't need to do anything.